orc8r/cloud/go/services/orchestrator/servicers/builder_servicer.go

```diff
@@ -191,3 +198,18 @@ func getVpnMconfig(gatewayConfig *models.MagmadGatewayConfigs) *mconfig_protos.O
 
 	return ret
 }
+
+func getStateMconfig(net configurator.Network, gwKey string) *mconfig_protos.State {
+	mconfigProto := &mconfig_protos.State{
+		SyncInterval: 60,
+		LogLevel:     protos.LogLevel_INFO,
+	}
+	netConfig := net.Configs["state_config"]
+	if netConfig != nil {
+		nsConfig := netConfig.(*models.StateConfig)
```

Comment:
is it possible for this cast to fail? what should happen in that case?

(right now, line 210 will panic because nsConfig will be nil if it's not a *models.StateConfig)